catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -901,23 +890,47 @@ public Response addDocument(
     return response;
   }
 
-  CreateInfo parseAttachments(List<Attachment> contentParts, String transformerParam) {
+  Pair<AttachmentInfo, Metacard> parseAttachments(
+      List<Attachment> contentParts, String transformerParam) {
 
     if (contentParts.size() == 1) {
       Attachment contentPart = contentParts.get(0);
-      return parseAttachment(contentPart);
+
+      InputStream attachmentInputStream = null;
+
+      try {
+        attachmentInputStream = contentPart.getDataHandler().getInputStream();
+      } catch (IOException e) {
+        LOGGER.info("IOException reading stream from file attachment in multipart body", e);
```

Comment:
Should we log the exception at the `info` level?

Reply:
changed to `debug`